### PR TITLE
Travis setup

### PR DESCRIPTION
**Pull request by mr-spod**

### CODE REVIEW

File: Podfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 target 'DrX' do
   use_frameworks!
-  podspec name: 'DrX.podspec'
+  podspec :name => 'DrX.podspec'
```
Comment (rwachowski) [inline Podfile]: Did this cause issues? This should be left as `name:` as it also resolves to a symbol. `:name =>` is "legacy"

File: Gemfile [hunk 2]
```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
+
+gem 'cocoapods', '~> 1.4.0'
```
Comment (rwachowski) [inline Gemfile]: Do you need still need this file?

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+xcode_workspace: DrX.xcworkspace #3
+xcode_scheme: DrX #4
+xcode_sdk: iphonesimulator11.2
+
+before_install:
+  - gem install cocoapods -v '1.4.0'
+
+install:
+  - pod repo update
+  - pod install
```
Comment (rwachowski) [inline .travis.yml]: could this step be `pod install --repo-update`?

**Review by rwachowski — COMMENTED** [review]
A few questions regarding the changes necessary to use Cocoapods v1.4.0

**Comment by mr-spod** [comment]
@rwachowski resolved your comments regarding cocoapods

**Comment by rwachowski** [comment]
lgtm

**Pull request merged**